README.md

```diff
@@ -116,43 +111,21 @@ To support the continued growth of the IPFS Network, we will be testing, benchma
 
 Our friends from libp2p will be playing the larger role into achieving this goal. If you are a Networks specialist and/or if you like scaling systems to serve hundreds of millions, join the conversation at [Bi-Weekly Call](https://github.com/libp2p/team-mgmt#bi-weekly-call) and learn more.
 
-### 4. Create a platform that untangles the mysteries of developing a DApp
-
-Support the growth of the DApp ecosystem with tooling, best practices, and community support.
-- Fully launch Peer Base toolkit to support building DApps that can run Online, Offline, and in Disjoint Networks.
-- Translate the Identity Manager RFC into code, providing a sandboxed way for Users to manage their DWeb Identity.
-- Support DApp security needs (API security, multi-tenancy, and secure broadcast updates).
-- Create a guidebook to popularize best practices to deal with unique UX challenges of the DWeb.
-- Continue to contribute to the adoption of the DWeb in the Web Platform (Brave, libdweb, IPFS companion), making it more accessible to users and Web Developers.
-
-The Dynamic Data & Capabilities WG is working full speed towards this. If you are passionate about building DApps and/or have field experience you would like to share, please come and participate in the [DDC Bi-weekly Sync](https://github.com/ipfs/dynamic-data-and-capabilities#bi-weekly-sync).
-
-### 5. The IPFS Community gets together for the 1st [IPFS Conf](https://github.com/ipfs/conf)
+### 4. The IPFS Community gets together for the 1st [IPFS Conf](https://github.com/ipfs/conf)
 
 Yes, you read the title right. The IPFS Community is going to organise the first "IPFS Conf" in the beautiful city of Lisbon. This will be an event for all the users and builders of the DWeb to come together to learn about the exciting progress that has been happening.
 - Host the 1st Conference for the IPFS Community.
 
 More details will be public soon, meanwhile, make plans to visit Lisbon in June! Subscribe for updates [here](https://github.com/ipfs/conf#ipfsconf-lisbon-2019).
 
-### 6. Everyone gets a simple way to add, share, retrieve and persist files and websites on the IPFS Network
-
-The primitives and the network are here. Now we need to create a great user experience for the DWeb by integrating it with the tools and types of services we use everyday.
-- Website Hosting on IPFS becomes simple and easy with migration paths from common sources (e.g. migrate or mirror Github Pages to IPFS).
-- The migration to Base32 CIDs is completed across all projects and IPFS Gateway.
```

Comment:
This is still important (and inflight) - does it fit into any of the other epics or another bucket of work?